src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -1069,16 +1077,24 @@ export default class ChargingStationService {
       // Check if user is authorized
       const user = await Authorizations.isAuthorizedToStartTransaction(req.user.tenantID, chargingStation, filteredRequest.args.tagID,
         ServerAction.CHARGING_STATION_REMOTE_START_TRANSACTION, Action.REMOTE_START_TRANSACTION);
-      if (!user.issuer) {
-        throw new AppError({
-          source: Constants.CENTRAL_SERVER,
-          errorCode: HTTPError.GENERAL_ERROR,
-          message: `User not issued by the organization execute command '${Command.REMOTE_START_TRANSACTION}'`,
-          module: MODULE_NAME, method: 'handleAction',
-          user: req.user,
-          action: action
-        });
+      if (!user.issuer) { // For testing purposes: authorize OICP User (Remote Start from Frondend)
```

Comment:
That's the kind of breaking change that needs to be done properly to allow the PR merge: 
The authorisation code need to be updated to cope with Hubject requirement and a user shall be returned only if a user has been found with the correct rights locally or on a roaming platform.

Reply:
I know that. I deleted it.

For local testing. That was the only way to authorize, since I have no physical Charging station here